frontend/packages/noobaa-storage-plugin/src/components/object-bucket-page/state.ts

```diff
@@ -0,0 +1,49 @@
+type State = {
+  name: string;
+  scName: string;
+  sizeValue: string;
+  sizeUnit: string;
+  progress: boolean;
+  error: string;
+  payload: Record<string, any>;
```

Comment:
this should be
```suggestion
  payload: Record<string, K8sResourceKind>;
```

right ?

Reply:
The whole payload is K8sResource kind. So I guess I should directly use K8sResourceKind.